specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2016-12-01/bms.json

```diff
@@ -5882,6 +5929,10 @@
             "name": "CreateMode",
             "modelAsString": true
           }
+        },
+        "vaultId": {
```

Comment:
why is vaultId needed here?  was it defaulted before - is this the same vaultId that is part of the path or is it a different vaultId